Clarify automerge requires bypass on all rulesets (#4743)
mintlify/docs: guides/configure-automerge.mdx

<Note>
  The Mintlify GitHub App must have bypass permissions on **every** ruleset that targets your deploy branch, including rulesets defined at the organization level and the repository level. If any ruleset is missing the bypass entry, the workflow creates a PR, but does not merge it.
</Note>

## Prerequisites

- Repository admin access (and organization admin access if there are org-level rulesets)
- [Mintlify GitHub App](/deploy/github) installed on the repository
- One or more rulesets targeting the `main` branch (at the org or repo level)

## Steps

Repeat these steps for every ruleset that applies to your deploy branch. Both **organization-level** and **repository-level** rulesets.

<Steps>
  <Step title="Navigate to rulesets">
    - **Repository rulesets**: On GitHub, go to your repository's **Settings** -> **Rules** -> **Rulesets**.
    - **Organization rulesets**: Go to your organization's **Settings** -> **Rules** -> **Rulesets**.
  </Step>
  <Step title="Select the ruleset">

>     Click a ruleset that targets your deploy branch (e.g. `main`).

Reviewer: ```suggestion
    Click a ruleset that targets your deploy branch, which is usually `main`.
```
